draft-ietf-masque-h3-datagram.md

```diff
@@ -250,8 +140,8 @@ corresponding stream.
 
 This specification introduces the Capsule Protocol. The Capsule Protocol is a
 sequence of type-length-value tuples that allows endpoints to reliably
-communicate request-related information end-to-end, even in the presence of
-HTTP intermediaries.
+communicate request-related information end-to-end on HTTP request streams, even
+in the presence of HTTP intermediaries.
 
 
 ## Capsule Protocol {#capsule-protocol}
```

Comment:
There is text missing from the introduction here that establishes the conditions on the use of capsule protocol.  That is, it only applies to those protocols that use extended CONNECT and also opt in to the use of the capsule protocol.

The nice thing with the draft up until this point is that it doesn't constrain the use of datagrams to those protocols that use extended CONNECT.  This part does.

(I know that this is perhaps fanciful, but I can imagine uses for regular HTTP requests that are enhanced by the use of DATAGRAM frames.)

Reply:
Agreed. I refactored the capsule section to contain more information up front.